cartridges/app_custom_pd/cartridge/experience/components/commerce_assets/categoryComponent.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "Custom Category Component",
+    "description": "A custom category component with specific settings.",
+    "group": "commerce_assets",
+    "attribute_definition_groups": [
+        {
+            "id": "customCategoryComponent",
+            "name": "Custom Category Component",
+            "description": "Settings for the custom category component.",
+            "attribute_definitions": [
+                {
+                    "id": "category",
+                    "name": "Category",
+                    "description": "Select Category to be displayed.",
+                    "type": "category",
+                    "required": true
+                },
+                {
+                    "id": "catDisplayName",
```

Comment:
Use better naming convention. You have 'catDisplayName' and 'categoryShortDescription' below